<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-04-29T13:04:51 IST" name="Default suite" finished-at="2023-04-29T13:05:05 IST" duration-ms="14137">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:runnerClass.RunnerIO@58d75e99]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2023-04-29T13:04:51 IST" name="Default test" finished-at="2023-04-29T13:05:05 IST" duration-ms="14137">
      <class name="runnerClass.RunnerIO">
        <test-method is-config="true" signature="setUpClass()[pri:0, instance:runnerClass.RunnerIO@58d75e99]" started-at="2023-04-29T13:04:51 IST" name="setUpClass" finished-at="2023-04-29T13:04:52 IST" duration-ms="477" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:runnerClass.RunnerIO@58d75e99]" started-at="2023-04-29T13:04:52 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2023-04-29T13:05:05 IST" duration-ms="13444" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Login to VTiger Application With Valid Credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Login To Vtiger Application]"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [vtiger CRM 5 - Commercial Open Source CRM] but found [Vtiger CRM 5 - Commercial Open Source CRM]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [vtiger CRM 5 - Commercial Open Source CRM] but found [Vtiger CRM 5 - Commercial Open Source CRM]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at stepDefinitions.LoginTest.validate_for_home_page_title(LoginTest.java:56)
at âœ½.Validate for home Page title(file:///D:/MavenProjects/BDD.FrameworkPractice/./src/test/java/featureFiles/Login.feature:10)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:runnerClass.RunnerIO@58d75e99]" started-at="2023-04-29T13:05:05 IST" name="tearDownClass" finished-at="2023-04-29T13:05:05 IST" duration-ms="160" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- runnerClass.RunnerIO -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
